docs/guides/01-getting-started/prerequisites.md

```diff
@@ -0,0 +1,204 @@
+
+### Install WebView2
+
+:::note
+WebView2 is pre-installed in Windows 11
+:::
+
+Tauri heavily depends on WebView2 to render web content on Windows, therefore you must have WebView2 installed. The easiest way is to download and run the Evergreen Bootstrapper from the [official website][download webview2].
+
+The bootstrapper script will try to determine the correct architecture and version for your system. Still, if you run into issues - especially with Windows on ARM - you can select the correct standalone installer or even a fixed version.
+
+## Setting Up macOS
+
+To install Rust on macOS, open a terminal and enter the following
+command:
+
+```console
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+```
+
+The command downloads a script and starts the installation of the `rustup` tool, which installs the latest stable version of Rust. You might be prompted for your password. If the installation was
+successful, the following line will appear:
+
+```text
+Rust is installed now. Great!
+```
+
+You also need to install CLang and macOS development dependencies. To do this, run the following command in your terminal:
+
+```console
+xcode-select --install
+```
+
+## Setting Up Linux
+
+To install Rust on Linux, open a terminal and enter the following command:
+
+```console
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+```
+
+The command downloads a script and starts the installation of the `rustup` tool, which installs the latest stable version of Rust. You might be prompted for your password. If the installation was successful, the following line will appear:
+
+```text
+Rust is installed now. Great!
+```
+
+You also need to install a couple of system dependencies, such as a C compiler and `webkit2gtk`. Below are commands for a few popular distributions.
+
+<Tabs>
+  <TabItem value="debian" label="Debian" default>
+
+```console
+sudo apt update
+sudo apt install libwebkit2gtk-4.0-dev \
+    build-essential \
+    curl \
+    wget \
+    libssl-dev \
+    libgtk-3-dev \
+    libappindicator3-dev \
+    librsvg2-dev
+```
+
+  </TabItem>
+  <TabItem value="arch" label="Arch">
+
+```console
+sudo pacman -Syu
+sudo pacman -S --needed \
+    webkit2gtk \
+    base-devel \
+    curl \
+    wget \
+    openssl \
+    appmenu-gtk-module \
+    gtk3 \
+    libappindicator-gtk3 \
+    librsvg \
+    libvips
+```
+
+  </TabItem>
+  <TabItem value="fedora" label="Fedora">
+
+```console
+sudo dnf check-update
+sudo dnf install webkit2gtk3-devel.x86_64 \
+    openssl-devel \
+    curl \
+    wget \
+    libappindicator-gtk3 \
+    librsvg2-devel
+sudo dnf group install "C Development Tools and Libraries"
+```
+
+  </TabItem>
+</Tabs>
+
+### Windows Subsystem for Linux (WSL)
```

Comment:
So the problem is, it's only accurate _in theory_. Like it's really just the luck of the draw if it works correctly or not; x servers on windows are just super hacky 🤷 

People reporting issues are somewhat common the last couple of days/weeks and it's never something tauri specific but just x server weirdness, i even recognize one of these issues from back when i used wsl a few years back (i wasn't able to figure it out back then and completely dropped wsl in my setup lol).

Reply:
Do we have enough info to write it _correctly_ now?

Mentioning WSL vs WSL2 vs WSLG, let's not forget that potential enterprise users might not have the luxury of upgrading to win 10 if that's a consideration ;)

Reply:
i don't think enterprise users (should) use these hacky x servers in the first place lol.

> Do we have enough info to write it correctly now?

Yes and no. In theory it's only missing one tiny thing (`LIBGL_ALWAYS_INDIRECT=1`) to match the broader range of wsl setup docs and it's one of multiple causes of the most common error message. Buuut x servers on windows, especially together with webkit(!) is so unstable that it's more often than not, not enough, it's also really dependent on the systems and stuff so it's super hard to debug it from our side.

**Again, my main problem here is that by providing docs for it we're suggesting that a) it's (well) supported and b) that we can provide help on issues.** Which is both not the case, which can be seen by the non-resolved issues (on discord) 😅 

P.S. I'm mostly afraid here because i'm like the only Windows user in the team😂 

P.P.S There's still hope that WSLg can make it into Windows 10 and i'm still praying everyday that MS follows through with it.

Reply:
P.P.P.S. Since i don't run _any_ VMs on Windows anymore i may try out wsl again and see if i stumble upon any common issues.

Reply:
I agree with @FabianLars this will cause us more problems that it is worth. Remember that this is the doc people will read. They will be excited about wsl and then quickly discouraged if things turn out badly

Reply:
Good with whatever way you guys decide. If we keep it then we can always add one of those warning call-outs saying it isn't officially supported

Reply:
So I'm hearing remove WSL, so the whole section under https://deploy-preview-590--tauri.netlify.app/v1/guides/getting-started/prerequisites#windows-subsystem-for-linux-wsl 
Do we want to keep the callout here on referring to linux instructions if using WSL https://deploy-preview-590--tauri.netlify.app/v1/guides/getting-started/prerequisites#setting-up-windows?

Reply:
nope, just make all the wsl go away 😂

Reply:
Will update this now ;)

Reply:
And done